tests/samples/schema.org/product.json

```diff
@@ -0,0 +1,19 @@
+[{"type": "http://schema.org/Product",
+  "properties": {"brand": "ACME",
+                 "name": "Executive Anvil",
+                 "image": "anvil_executive.jpg",
+                 "description": "Sleeker than ACME's Classic Anvil, the\n      Executive Anvil is perfect for the business traveler\n      looking for something to drop from a height.",
```

Comment:
It looks suspicious, as whitespace in the beginning and at the end seems to be stripped from the value. But this is not caused by this PR.